src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -20,15 +35,18 @@ function transformApplicants(applicants) {
   applicants.forEach(app => {
     const transformedApp = {
       fullName: app.applicantName ?? '',
-      sshOrTin: app.applicantSSN ?? '',
-      dateOfBirth: app.applicantDOB ?? '',
+      ssnOrTin: app.applicantSSN?.ssn ?? '',
+      dateOfBirth: fmtDate(app.applicantDOB) ?? '',
```

Comment:
fix for SSN being nested and the date of birth needing to have the digits swapped